index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Panda_login</title>
+  <link rel="icon" href="images/logo/favicon.ico">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  
+  <div class="container">
+    <div class="logo"><a href="/">
+    <img src="images/logo/logo.svg" width="396px" height="132px"></a>
```

Comment:
❗️ 수정요청
이미지 태그의 width, height 속성은 단위없는 정수를 입력해야합니다~ 즉 px같은 고정형 단위의 정수값만 입력이 가능합니다.
따라서 반응형 단위 사용을 원하시면 css로 해주세요!

[mdn](https://developer.mozilla.org/ko/docs/Web/HTML/Element/img#width)

```suggestion
    <img src="images/logo/logo.svg" width="396" height="132"></a>
```